script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -74,27 +73,27 @@ And what remains is how it's always been, the usua――
 [sha:947955f32c48b5704f4ad363bd1f33572a36e17d]{
 -- Page 36, Offset 14896.
 -- 今まで通りの、先輩の姿があった。
-――the usual sight of Senpai...
+―――Everyday sight of Senpai...
 }
 [sha:7283bb3fdcb5d4f90baf354cd0950f752a47a7f2]{
 -- Page 37, Offset 14897.
 -- 「ちょっ、先輩!?　なんで<学|こ><校|こ>に!?」
-"Wait, Senpai!? Why are you still here?!"
+"Wait, Senpai?! What are you doing here?!"
 }
 [sha:98a9c8cef1c8af51b7ecc2f92b16c68e5c733f0c]{
 -- Page 37, Offset 14898.
 -- 「なんでと言われましても、わたし、ここの学生ですし。
-"Why are you asking me why? I'm a student here.
+"What am I doing here? I mean, I'm a student...
```

Comment:
Somehow this one just feels so much more boring now.

Reply:
Prefer first sentence of this ver. and second half of old version.
"What am I doing here? I'm a student [here]."
"here" optional because repetition